app/subscriber/src/components/sidebar/hooks/useFilterOptions.ts

```diff
@@ -1,14 +1,32 @@
-import { useLookup } from 'store/hooks';
+import React from 'react';
+import { useSources } from 'store/hooks/subscriber/useSources';
+import { ISourceModel } from 'tno-core';
 
 export const useFilterOptions = () => {
-  const [{ sources }] = useLookup();
-  const dailyPrint = sources.filter(
-    (source) =>
-      source.code === 'SUN' ||
-      source.code === 'PROVINCE' ||
-      source.code === 'TC' ||
-      source.code === 'GLOBE' ||
-      source.code === 'POST',
-  );
-  return { dailyPrint: dailyPrint, sources };
+  const [, api] = useSources();
+  const [sources, setSources] = React.useState<ISourceModel[]>([]);
+
+  React.useEffect(() => {
+    api.findAllSources().then((res) => setSources(res));
+    // only run once
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  const dailyPrint = sources.filter((source) => source.productId === 1);
```

Comment:
Hardcoding product ids is a good way for this not to work in any other environment.  You have to extract hardcoded values and place them into the settings.  The settings is a freeform `value` so you could add JSON as a value and enter the appropriate values there.